src/twisted/protocols/sip.py

```diff
@@ -754,76 +828,104 @@ def sendMessage(self, destURL, message):
             raise RuntimeError("only UDP currently supported")
         if self.debug:
             log.msg("Sending %r to %r" % (message.toString(), destURL))
-        self.transport.write(message.toString(), (destURL.host, destURL.port or self.PORT))
+        data = message.toString()
+        if isinstance(data, unicode):
+            data = data.encode("utf-8")
```

Comment:
Is it OK to assume message.toString() will always return a unicode string that we can encode into utf-8? The doc string doesn't specify what assumptions it makes on message